pkgs/applications/science/electronics/kicad/versions.nix

```diff
@@ -25,23 +25,23 @@
   };
   "kicad-unstable" = {
```

Comment:
Could you also change the pname and versions so that they would follow the naming scheme, as outlined in the [docs](https://github.com/NixOS/nixpkgs/blob/master/doc/contributing/coding-conventions.chapter.md#package-naming-sec-package-naming)? As per #68518.



> If a package is not a release but a commit from a repository, then the version attribute must be the date of that (fetched) commit. The date must be in "unstable-YYYY-MM-DD" format.

Reply:
Scratch that, I didn't realize there are two sets of this package. :(